

"""
当程序运行检测到一个错误时  ，python解释器无法继续执行了 出现了一些错误的提示  这就是所谓的异常，就是我们常说的BUG

"""

# 读取一个不存在的文件就会导致异常
# f = open("d:/abc.txt", "r", encoding="utf-8")